Formula/apachetop.rb

```diff
@@ -28,11 +28,15 @@ def install
                           "--with-logfile=/var/log/apache2/access_log"
     system "make", "install"
   end
+
+  test do
+    assert_match "ApacheTop v#{version} - Usage:", shell_output("script -q /dev/null #{bin}/apachetop -h | cat")
+  end
 end
 
 __END__
 --- src/resolver.h    2005-10-15 18:10:01.000000000 +0200
-+++ src/resolver.h        2007-02-17 11:24:37.000000000 
```

Comment:
No whitespace changes please

Reply:
Atom is removing the white spaces. I'll take a look on it.